pkgs/applications/science/molecular-dynamics/viennarna/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchurl
+, gsl
+, mpfr
+, perl
+, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ViennaRNA";
+  version = "2.4.17";
+
+  src = fetchurl {
+    url = "https://www.tbi.univie.ac.at/RNA/download/sourcecode/2_4_x/${pname}-${version}.tar.gz";
+    sha256 = "08f1h2a8fn1s2zwf1244smiydhgwxgcnzy6irpdlmpvwygv0irmi";
+  };
+
+  buildInputs = [
+    gsl
+    mpfr
+    perl
```

Comment:
Did you check if this is belongs into nativeBuildInputs?

Reply:
How? Should I just try it and test whether it builds with `perl` moved to `nativeBuildInputs`? Should I also try to move `python3`?

Reply:
I think these really belong to `buildInputs` because the created Perl/Python modules are binary

Reply:
if anything is in PATH and is run at build time it is native but if you tested it is fine.